Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -87,6 +87,10 @@ def genGridOnPoints(data1,gm,deep=True,grid=None,geo=None):
     if grid is None:
       x = g.getLongitude()[:]
       y = g.getLatitude()[:]
+      xm = x[0]
+      xM = x[-1]
+      ym = y[0]
```

Comment:
and that note, @dlonie did a great job renaming variables when reformatting. So now I don't recognize my own code :stuck_out_tongue: but it IS better, just need to learn all the new names...